tutorials/Policy_Gradient_Vanilla.jl

```diff
@@ -0,0 +1,94 @@
+using Flux, CuArrays
+using OpenAIGym
+import Reinforce.action
+import Flux.params
+using CUDAnative: log
+using Flux: onehot
+using Statistics
+
+#-------Define Custom Policy------------
+mutable struct CartPolePolicy <: Reinforce.AbstractPolicy
+  train::Bool
+
+  function CartPolePolicy(train = true)
+    new(train)
+  end
+end
+
+#----------Setup Environment-------
+env = GymEnv("CartPole-v0")
+
+#----------Define Hyperparameters-------
+STATE_SIZE = length(env.state)
+ACTION_SIZE = length(env.actions)
+γ = 1.00f0    # discount rate
+
+η = 1e-3 # Learning rate
+
+#---------_Define Model And Loss Function---------
+policy = Chain(Dense(STATE_SIZE,30,relu),Dense(30,50,relu),Dense(50,ACTION_SIZE))
+
+function loss(s,actions,A_t)
+    pi = softmax(policy(s))
+    logpi = log.(mean(pi .* actions,dims=1) .+ 1f-10)
+    return mean(-logpi .* A_t)
+end
+
+opt = ADAM(params(policy),η)
```

Comment:
`opt = ADAM(params(policy),η)` could be
`opt = ADAM(η)` in accordance with the new optimizer API